.travis.yml

```diff
@@ -88,13 +88,13 @@ cache:
 
 env:
   global:
-    # encrypt with: travis encrypt WHITESOURCE_PASSWORD=... --com
-    - secure: "uUebwKsw2RWsXkzQxFkbYR5YpYTKlFkPUpczAqditDK3JO6VHTex09py262MEtjOVUu1wx1aw/0E9RUE0FnAG9jVHlDY1EuuvfGTAACh/uuAuyVOm8utE0TCnEzSEvhyRydBqq8Kb+9wJfjsocCeTY+aSapkWJFpEPROJi5Qpx7jReV0drGuxLw3QlC5gEn6vq9oy2QwhMe5iLQd2TwBSNQ9V2StAO/fiYwJI64BaFtw3e4SLSZJ0kyIUPpmj0u2BWPqRP4PKCnNTjrIvi21sVX+SxHUzlb2YMOnwFJx12le89YR1LeQTbuF/tmMDha2ArMeGcZEqQ1pCnDrMaa5RbHH5iH/QJEBFTU86bx1t+NuldVxqC8IA9ymVnXYiaMwM5OBNnpORXlyZQX8sxsQ+4xXe9scwjIbYmHyr/pqqy7vunhgsu6z1jUk739KNNC0/6f7MawgB6BbvlDMIvAjOFIkwWxe7dsgvmaqKhZjC4bi2Af655g5aTLPE4JKT9gQBrVFLFeLZWzJDv81iLUrC/vQbT6b0+yZ5byoPFxNB80FeT07I9qG9PBQF4NAYTVR1p9EEEusEfmrP+FBy3WCgIQcEON+5XIqD2kUzsxBEGX2hGVcg5FdEhfKen7gPYl5NbfxSXYKxjSvKCqTuFCgRh2N0aa1uyE4AGV66jSMo3E="
-    # encrypt with: travis encrypt DEPLOY_PASSPHRASE=... --com
+    # encrypt with: travis encrypt --pro WHITESOURCE_PASSWORD=... 
+    - secure: "InIVb2M8beewAql4wgqsnOnlMNFpJf9uRVuaSZG8m0lIJGsF9XvciqUZiLyPDMjPtXvnsRNaAAe+EzboQ/fdu8yAN8ZrnR3qu1AWZckyY5wI/ycAyxSFW8RwaLcKidnIYbda3jHce3iNSPcujy/2qPEx/isPvw799BYmkHsPD4FXUMVa+eC6rgv686HzWqkIBd9dYho9GCuXWfrd68GlwhNjgjEIytfGGrm6b6+105do/D+HycQ1+Q9tfoeUYOW3IEtoX8W9z+c9NeFxSxfo45rR94F/Omxt8XYjzTjHdZLjxW0r+cRjTK2/zxZFerctZlCH8WPpNzZ0d1O0b1uLiezUGy2E/LDezH+ZYezByMbgCEBRwXE0xxJ7iInHDeqameacBqjwgAeK+MxE5YYEpNRCh601WE6dO2v74iGpnFm4eKLIvvQazVKsRzAED3DK6R8QBOjj232h6xyCYrpEsKn8b/7vwy3tepMgicjWqFYsMX+sNVOfAKUkiBt/pS88mOr6DICA5le39Vvwq5f0UHflsu+RG2zDRpsnMaxicdA+WXsCFYjSBiF6ukY8iGVLByjGjAUTbKaoVO7zfql87R0MuqyzkPmysbhcX3zJSvb2d+M/ZLryUNdSArog3sAI9TU8tZWvWsDuGTnH28/vJ5cvqbruKry441ODv5g8v28="
+    # encrypt with: travis encrypt --pro DEPLOY_PASSPHRASE=...
```

Comment:
I don't think the position of the `--pro` vs `--com` makes any difference. My tests only listed the WHITESOURCE_PASSWORD as being wrong. I've sync'd all coments just because.